fabryca_frontend/src/components/ProjectForm.js

```diff
@@ -0,0 +1,54 @@
+import React, {useEffect, useState} from 'react';
+import { Frame, Input, Button} from '@react95/core'
+import './TicketForm.css';
+import { useNavigate } from 'react-router-dom';
+import CursorButton from './CursorButton';
+
+
+ const TicketForm = () =>{
+    const [name, setName] = useState('');
+
+  
+    const postToDatabase = async (newProject) => {
+
+      const body = JSON.stringify(newProject);
+
+      await fetch('https://fabrycaapi.azurewebsites.net/api/Projects',{
+        method: 'POST',
+        mode: 'cors',
+        accept: 'text/plain',
+        headers:{'Content-Type':'application/json'},
+        body: body
+      }).then(r=>r.json()).then(res=>{
+        if(res){
+          console.log(res);
+        }
+      });
+  
+    }
+    const navigate = useNavigate();
+    const handleSubmit = e => {
+      e.preventDefault();
+      const newProject={
+        name: name
+        }
+        
+      postToDatabase(newProject);
+
+      navigate('/projects');
+  
+      setName('');
+    }
+
+    return(
+      <Frame className='ticket__form__frame'>
+        <form onSubmit={handleSubmit} className="ticket__form">
```

Comment:
Bad! This will give us problems if we try to style the ticket form and inadvertedly change the project form